actions_helper/main.py

```diff
@@ -28,6 +29,22 @@ def cmd_wait_for_service_stability(
     )
 
 
+@cli.command(name="wait-for-task-stopped")
```

Comment:
This function seems to be very similar with `wait-for-service-stability`. Could we share the code?

Reply:
There are not much similarity, I will prefer to keep it separate as it is.
Unless we want to share the config, which i will prefer not to, because there use cases are different